src/harness/reference_models/pre_iap_filtering/pre_iap_util.py

```diff
@@ -0,0 +1,186 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import shapely.geometry as sgeo
+from reference_models.geo import vincenty
+from sas_test_harness import generateCbsdReferenceId
+from reference_models.geo import utils
+
+FSS_GWBL_PROTECTION_DISTANCE = 150
+
+
+def purgeOverlappingGrants(cbsds, frequency_range):
+  """Removes Grants from CBSDs that overlap with the given frequency range.
+
+  Args:
+  cbsds: List of CbsdData objects. Grants are removed from the input.
+  frequency_range: A dictionary with keys 'lowFrequency' and 'highFrequency'
+  """
+  for cbsd in cbsds:
+    if not cbsd['grants']:
```

Comment:
Most likely this test is useless and does not bring much, since the following list comprehension will quickly return an empty list as well.

Reply:
Removed   the check